packages/common/src/initiative-templates/_internal/InitiativeTemplateUiSchemaEdit.ts

```diff
@@ -89,6 +95,111 @@ export const buildUiSchema = async (
           },
         },
       },
+      {
+        type: "Control",
+        scope: "/properties/recommendedTemplates",
+        labelKey: `${i18nScope}.fields.recommendedTemplates.label`,
+        options: {
+          control: "hub-field-input-gallery-picker",
+          targetEntity: "item",
+          catalogs: getRecommendedTemplatesCatalog(
+            context.currentUser,
+            i18nScope
+          ),
+          facets: [
+            {
+              label: `{{${i18nScope}.fields.recommendedTemplates.facets.sharing:translate}}`,
+              key: "access",
+              field: "access",
+              display: "multi-select",
+              operation: "OR",
+            },
+          ],
+          drag: false,
+          linkTarget: "workspaceRelative",
+        },
+      },
     ],
   };
 };
+
+const getRecommendedTemplatesCatalog = (user: IUser, i18nScope: string) => {
```

Comment:
I'd maybe suggest moving these to be `_internal` utils rather than defining it directly in this file?
1. Keeps this (already slightly overwhelming) file a bit cleaner 
2. Allows you to mock the function/return value in the corresponding `InitiativeTemplateUiSchemaEdit.test.ts` so it also doesn't get so verbose. 

But this isn't necessary if you want to keep it here.